developer/src/kmcmpdll/Compiler.cpp

```diff
@@ -293,7 +293,7 @@ KMX_BOOL AddCompileMessage(KMX_DWORD msg)
 
   if (szTextp) {
     strcpy(szText, szTextp);
-    strcat(szText, ending); }
+    strcat(szText, ending);}
```

Comment:
Closing brace should go one separate line unless the opening brace is in the same line:

```suggestion
    strcat(szText, ending);
  }
```

Reply:
👍